skyportal/handlers/api/candidate.py

```diff
@@ -411,10 +415,10 @@ def get(self, obj_id=None):
             "undefined",
         ]:
             start_date = arrow.get(start_date).datetime
-            q = q.filter(Obj.last_detected >= start_date)
+            q = q.filter(Candidate.passed_at >= start_date)
```

Comment:
I think it should go in to this PR as this new line is filtering on `passed_at`, which, without an index, could become a bottleneck

Reply:
@dannygoldstein it was previously filtering on an unindexed property, so no change there

Reply:
If you insist, please file a PR against this branch

Reply:
done